pages/[[...swingset]].jsx

```diff
@@ -126,6 +128,9 @@ const components = {
   FormikStateViewer,
   NextRouterDebugger,
   ComboboxField, // @TODO - Consider Swingset support for components at nested entry points
+  Toaster,
+  Notification,
```

Comment:
You can specify `peerComponents` in docs.mdx to avoid having to pass this in (see suggestion in `docs.mdx`)

(this would require `Notification` to have a `docs.mdx` file)

Reply:
👍🏼  removed updates to the swingset config.